### PR TITLE
Change typography fonts

### PR DESCRIPTION
**Pull request by cariocadev**
@wkurosawa i think that project is organized but if you have more any idea, please tell me.

### CODE REVIEW

File: metronic/sass/global/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		url('../fonts/Gotham-Medium.woff2') format('woff2'),
+		url('../fonts/Gotham-Medium.woff') format('woff'),
+		url('../fonts/Gotham-Medium.ttf') format('truetype'),
+		url('../fonts/Gotham-Medium.svg#Gotham-Medium') format('svg');
+	font-weight: 500;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'Gotham';
```
Comment (wkurosawa) [inline metronic/sass/global/_fonts.scss]: @cariocadev is it ok to use the same `font-family` name to Gotham Medium and Gotham Book? Shouldn't it be different?

File: metronic/sass/layouts/layout4/layout.scss [hunk 2]
```diff
@@ -14,11 +16,11 @@
 @import 'layout/_footer';   // page footer
 @import 'layout/_theme-panel';  // theme customizer panel
 @import 'layout/_form';
-@import 'layout/_inputs'; 
+@import 'layout/_inputs';
+
```
Comment (wkurosawa) [inline metronic/sass/layouts/layout4/layout.scss]: @cariocadev empty line?

File: metronic/sass/layouts/layout4/layout.scss [hunk 3]
```diff
@@ -14,11 +16,11 @@
...
 @import 'layout/_theme-panel';  // theme customizer panel
 @import 'layout/_form';
-@import 'layout/_inputs'; 
+@import 'layout/_inputs';
+
 
-
 @import '../quick-sidebar/_quick-sidebar';  // quick sidebar content
-
+ // necessary
```
Comment (wkurosawa) [inline metronic/sass/layouts/layout4/layout.scss]: @cariocadev is this comment necessary? Ahn, it's what's written but.. does it break without it?

File: metronic/sass/global/_fonts.scss (hunk 1 above)
Comment (cariocadev) [inline metronic/sass/global/_fonts.scss]: The difference is the font-weight.

Comment (wkurosawa) [inline metronic/sass/global/_fonts.scss]: @cariocadev Yes, I know, but you named Gotham Medium and Gotham Book using same font-family, `Gotham`. 
I think that the last one overwrites the first. Then, if you want to use 2 different families you should name one differently, such as: 
```
font-family: 'Gotham Book'
```
And set properly family when needed.

Comment (cariocadev) [inline metronic/sass/global/_fonts.scss]: Okay.

File: metronic/sass/layouts/layout4/layout.scss (hunk 3 above)
Comment (cariocadev) [inline metronic/sass/layouts/layout4/layout.scss]: I deleted this commentary.

**Comment by wkurosawa** [comment]
@cariocadev This PR is OK to be closed & merged!

**Pull request merged**